nemo/collections/tts/data/text_to_speech_dataset.py

```diff
@@ -78,8 +71,8 @@ class TextToSpeechDataset(Dataset):
         featurizers: Optional, list of featurizers to load feature data from. Should be the same config provided
             when running scripts.dataset_processing.tts.compute_features.py before training.
         feature_processors: Optional, list of feature processors to run on training examples.
-        align_prior_config: Optional, if provided alignment prior will be calculated and included in
-            batch output.
+        align_prior_hop_length: Optional int, hop length of audio features.
+            If provided alignment prior will be calculated and included in batch output.
```

Comment:
Could you please also add a comment on how it usually is equal to `hop_lengh` of spectrogram STFT?

Wouldn't hurt for this to be explicitly stated somewhere outside the default config.

Reply:
Added. Mentioned it as feature hop_length as we may not always be using STFT based features.

Reply:
Yes, thank you, that makes more sense